src/graphql/main/schema.graphql

```diff
@@ -964,7 +966,13 @@ type Query {
     targetConfirmations: TargetConfirmations = 1
     walletId: WalletId!
   ): OnChainTxFee!
-  onChainUsdTxFee(
+  onChainUsdFee(
```

Comment:
This is to get the fee for an invoice that is denominated in sats but paid from a usd wallet.